changelogs/unreleased/5004-constraint-conflict-std.yml

```diff
@@ -0,0 +1,4 @@
+description: "constraints Jinja and email_validator to same version as std"
+change-type: patch
+destination-branches:
```

Comment:
Should we add a bugfix entry?

Reply:
I would only add a bugfix entry if it ever broke something, but did it?

Reply:
[It did](https://inmanta.slack.com/archives/CKRF0C8R3/p1666012217831119). Only for Python>3.6 though (see [here](https://inmanta.slack.com/archives/CKRF0C8R3/p1666016740737439?thread_ts=1666012217.831119&cid=CKRF0C8R3)).